code/modules/mob/living/carbon/carbon.dm

```diff
@@ -85,6 +85,12 @@
 
 	return ..()
 
+/mob/living/carbon/CtrlShiftClick(mob/user)
+	..()
+	if(iscarbon(user))
+		var/mob/living/carbon/carbon_user = user
+		carbon_user.give(src)
```

Comment:
Put this on right click instead, have it only work if they're holding an item and they're not in combat mode

Reply:
Doesn't this draw with a syringe? There are probably other right-click non-combat mode interactions I can think of. I don't think Mothblocks would approve of blocking off that interaction for all items.

Reply:
ctrl + shift + click is a bit of an annoying combo, are you sure you can't just do shift + right click?

Reply:
shift right click opens the context menu

Reply:
> shift right click opens the context menu

ah yeah, that's right, alright, whatever I suppose this is fine until an alternative can be found